src/context.tsx

```diff
@@ -153,6 +156,7 @@ const reducer = (state: State, action: Record<string, any>): State => {
       pageState.filters[filterType as FilterType] = new Set(filterValue);
       return newState;
     };
+    // Note: deprecated but leaving here in case we ever need again
     case "UPDATE_META_ANALYSIS": {
```

Comment:
We have these in version control if needed, is it still worth to keep in our code?

Reply:
> ![image](https://user-images.githubusercontent.com/33983945/109402679-aee62000-7925-11eb-9cf1-44a03b078779.png)
> 
> For the mobile view, could the second graph go below the first graph instead of being side by side?

Mobile responsiveness is handled by tableau so tagging @jordanvanwyk